config/tsconfig.json

```diff
@@ -16,6 +16,6 @@
         "typeRoots": ["../src/typings"],
         "types": ["jasmine", "coveo-search-ui", "coveo-search-ui-tests"]
     },
-    "include": ["../src/**/*.ts", "../src/**/*.svg", "../src/**/*.json"],
+    "include": ["../src/**/*.ts", "../img/**/*.svg", "../src/**/*.json"],
```

Comment:
Je suis currieux de vois le code ressemble a quoi une fois transpiller ?

Reply:
`this.buttonElement = Object(coveo_search_ui__WEBPACK_IMPORTED_MODULE_0__["$$"])('div', {}, _utils_icons__WEBPACK_IMPORTED_MODULE_1__["paperclipIcon"]).el;`
`this.buttonElement = Object(coveo_search_ui__WEBPACK_IMPORTED_MODULE_0__["$$"])('div', {}, _img_paperclip_svg__WEBPACK_IMPORTED_MODULE_1__).el;`

The webpack shenanigans behind are pretty similar, just the hoop through utils before the svg markup as a string.

Reply:
Good 👍